docs/dummyvm/src/object_model.rs

```diff
@@ -0,0 +1,93 @@
+use crate::DummyVM;
+use mmtk::util::copy::{CopySemantics, GCWorkerCopyContext};
+use mmtk::util::{Address, ObjectReference};
+use mmtk::vm::*;
+
+pub struct VMObjectModel {}
+
+// This is the offset from the allocation result to the object reference for the object.
+// Many methods like `address_to_ref` and `ref_to_address` use this constant.
+// For bindings that this offset is not a constant, you can implement the calculation in the methods, and
+// remove this constant.
+pub const OBJECT_REF_OFFSET: usize = 4;
```

Comment:
Like all the different kinds of slots, this non-zero and unaligned value was introduced for testing the VO bits.  Few real-world VMs use this value.  We may set it to zero, and comment that it can be set to a different value if the ObjectReference in the VM has an offset from the starting address.

Reply:
Done. The default value is 0 with comments to explain it.